cmd/hack/hack.go

```diff
@@ -836,23 +764,25 @@ func testStartup() {
 func testResolve() {
 	startTime := time.Now()
 	//ethDb, err := ethdb.NewBoltDatabase("/home/akhounov/.ethereum/geth/chaindata")
-	//ethDb, err := ethdb.NewBoltDatabase("/Users/alexeyakhunov/Library/Ethereum/geth/chaindata")
-	ethDb, err := ethdb.NewBoltDatabase("statedb")
+	ethDb, err := ethdb.NewBoltDatabase("/Users/alexeyakhunov/Library/Ethereum/geth/chaindata")
```

Comment:
I think we need to change the path.

Reply:
Yes, it is my "hacking" toolkit, which I think will need to be transformed into a proper debugging toolkit

Reply:
Actually, I will also add more tests for these things rather